feg/gateway/services/s6a_proxy/servicers/s6a_proxy_test.go

```diff
@@ -217,10 +217,10 @@ func TestS6aProxyServiceWitPLMNlist(t *testing.T) {
 		}
 
 		// AIR
-		r, err := c.AuthenticationInformation(context.Background(), req)
-		if err != nil {
-			t.Fatalf("GRPC AIR with PLMN IMSI1 Error: %v", err)
-			complChan <- err
+		r, err2 := c.AuthenticationInformation(context.Background(), req)
```

Comment:
should we do `authenticationErr` instead of `err2`? Seems more descriptive

Reply:
sure, err2 is ugly